Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM golang:1.14 AS build
+FROM registry.redhat.io/ubi8/ubi AS build
 WORKDIR /go/src/app
+RUN yum -y install golang make
 COPY go.mod .
 COPY go.sum .
 RUN go mod download
 COPY . .
 RUN make
 
-FROM scratch
+FROM registry.redhat.io/ubi8/ubi
```

Comment:
Is the image bigger with that ? We can maybe neglect it.

Reply:
Yes the image is much bigger (looking at `podman images`), 7.45 MB VS 216 MB. And I did not realize that `scratch` was an empty FS, I thought this was some non-empty lean image. The idea was to use the same base image as all other containers in the cluster (crc dnsmasq image, openshift containers, ...) to maximize sharing. However, `registry.redhat.io/ubi8/ubi` is going to be a moving target, so there is some complexity involved if we want the same base image as openshift, and when you take into account OKD, you want yet another base image. I'll drop that part of the PR, keeping scratch seems easier :)